src/olympia/blocklist/migrations/0038_auto_20241112_1606.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 4.2.16 on 2024-11-12 16:06
+
+from django.db import migrations
+
+from olympia.core.db.migrations import CreateWaffleSwitch
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blocklist', '0037_alter_blockversion_block_type_and_more'),
+    ]
+
+    operations = [
+        CreateWaffleSwitch('soft-blocking-enabled'),
```

Comment:
I would call that `enable-soft-blocking` for consistency with _most_ other waffle switches governing features that we already have.